tests/test_config.py

```diff
@@ -59,3 +60,46 @@ def test_remove_password(self):
             expected = url.replace("PASS", replace_with)
             removed = remove_password_from_url(url, replace_with)
             self.assertEqual(removed, expected)
+
```

Comment:
This is stuff that was added recently to data-diff master, it is not intended as part of my proposed change.